templates/quickstart-confluence-master.template.yaml

```diff
@@ -1907,6 +1913,36 @@ Resources:
       FromPort: -1
       ToPort: -1
       SourceSecurityGroupId: !Ref SecurityGroup
+  EncryptionKey:
+    Condition: UseEncryptionAtRest
+    DeletionPolicy: Retain
```

Comment:
Should we add documentation that this resource is retained and you may want to clean up your keys if you regularly create and destroy stacks? Are there any other retained resources that could come under this umbrella?